src/common/drpc.c

```diff
@@ -133,35 +143,65 @@ drpc_response_create(Drpc__Call *call)
 void
 drpc_response_free(Drpc__Response *resp)
 {
-	drpc__response__free_unpacked(resp, &alloc);
+	struct drpc_alloc alloc = PROTO_ALLOCATOR_INIT(alloc);
+
+	drpc__response__free_unpacked(resp, &alloc.alloc);
 }
 
-static struct unixcomm *
-new_unixcomm_socket(int flags)
+static int
+unixcomm_close(struct unixcomm *handle)
+{
+	int ret;
+	int fd;
+
+	if (!handle)
+		return -DER_SUCCESS;
```

Comment:
I don't think there's a convention across the whole codebase, my preference is not to use zero to make it clear it's not a system error number, but consistently with the rest of code in the file wins out, will change.

Reply:
Ah. I recall being told directly that we ought to use 0 instead of DER_SUCCESS on some early reviews in this area. Only reason I brought it up. Doesn't seem to be mentioned in our style guidelines, though.